src/styles/utils/reset.scss

```
@@ -0,0 +1,9 @@
+* {
+  font-family: "Roboto", sans-serif;
+  font-size: 10px;
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+}
```

```suggestion
* {
  box-sizing: border-box;
}

body {
  margin: 0;
  font-family: "Roboto", sans-serif;
  font-size: 10px;
}
```
FYI: the proper way to do it. it's better not to set any properties directly in `*` selector except `box-sizing` (a lot of them might be inherited, some of them not ok to set for all elements, also it may affect performance)